src/components/Link.test.js

```diff
@@ -104,4 +104,18 @@ describe("Link", () => {
 
     expect(container.firstChild).toHaveAttribute("data-testid", "my-link");
   });
+
+  it("with analyticsClassName", () => {
+    const { container } = render(
+      <Link
+        href="/terms"
+        newTab={false}
+        analyticsClassName="trackcustomanalytics"
+      >
+        Terms and Conditions
+      </Link>
+    );
+
+    expect(container.firstChild).toHaveClass("trackcustomanalytics");
```

Comment:
The intent here is that the link itself gets the `className`, not necessarily container's first child.
Let's do what we did in the tests above:
```
const link = getByText("Terms and Conditions");

expect(link).toHaveClass("trackcustomanalytics");
```